### DISCUSSION TITLE
Latest Stable Version

### DISCUSSION THREAD

**Discussion by ch-ricardor** [ihsn/nada#47] [Q&A]
Hello,

I tested NADA few years ago, now I'm trying to run new tests.

Which is the latest stable version? 

The RDF upload has some issues in several versions, and the messages appear in other pages.  

The xml upload works.

I tested 5.0.4, 5.06, 5.2, Master

I need a starting point before I jump into the code.

I'm testing with php 8.0 and the try 8.1

Example:

```
<div class="alert alert-danger"><p>You did not select a file to upload.</p></div>

<div class="alert alert-success">PDF report generated successfully.datafiles/central/0bc04b5f2d560725c16d73a04e3f95f9/ddi-documentation-english-1.pdf</div>
``` 

![nadaCatalog](https://user-images.githubusercontent.com/22514626/194916921-71055997-77fc-4fb7-a23a-8b6494fd38ed.JPG)

**Comment by mah0001** [comment]
@ch-ricardor, Thank you for testing. The latest version is 5.2 (master). For the error message "You did not select a file upload", can you check the permissions for the datafiles folder (default is your-nada-folder/datafiles), you can try by temporarily setting the permissions to 777 to see if the error goes away.

**Comment by ch-ricardor** [comment]
Hello Mehmood

I'm reviewing the code to try to understand the concept. In this specific
case is question of validations in the form.

Looks like some error messages persist between screens. I'm reviewing the
upload  in the controller.

I'm making some changes, to understand the logic of the upload process.

By now I'm uploading from files created in nesstar publisher.

I don't know if someone review the issues posted in github, or how people
is collaborating.

Best regards

 Ricardo
